Documentation/using-rkt-with-systemd.md

```diff
@@ -157,42 +157,41 @@ To make socket activation work, add a [socket-activated port][aci-socketActivate
 }
 ```
 
-Then you will need a pair of `.service` and `.socket` unit files. The socket unit file will have the same port you've set in the image manifest of your app, and the service unit file will run `rkt`:
+Then you will need a pair of `.service` and `.socket` unit files. Let's use template unit files to make it more flexible:
 
 ```
-# my-socket-activated-app.socket
+# my-socket-activated-app@.socket
 [Unit]
 Description=My socket-activated app's socket
 
 [Socket]
-ListenStream=8080
+ListenStream=%i
 ```
 
 ```
-# my-socket-activated-app.service
+# my-socket-activated-app@.service
 [Unit]
 Description=My socket-activated app
 
 [Service]
-ExecStart=/usr/bin/rkt run myapp.com/my-socket-activated-app:v1.0
+ExecStart=/usr/bin/rkt run --port 80-tcp:%i myapp.com/my-socket-activated-app:v1.0
 KillMode=mixed
 ```
 
-Finally, start the socket unit:
+Finally, instantiate the socket unit. In this case, with port 8080:
```

Comment:
It's confusing and a bit magical why the port is different here - could do with some more detailed explanation of how 80 is the default in the aci, but can be overridden,etc